lag/lag.yaml

```diff
@@ -29,6 +29,13 @@ components:
         name:
           x-include: ../common/common.yaml#/components/schemas/Named.Object/properties/name
           x-field-uid: 4
+        lldps:
```

Comment:
singular - lldp

Reply:
added lldps as a array and port name as a key,as lldp can be configured for each of the lag ports in the lag,moved the configuration to lag level from lag port as in future if we open up more config elements and if required lag level globals for lldp, then we would not be able to do in lag port level